src/List/ItemList.js

```diff
@@ -22,22 +24,48 @@ const Items = props => {
     }
   }, [token, setdbItems, dbItems, props]);
 
-  //This will update an item to include a purchase date
-  const handleChange = e => {
-    let datePurchased = new Date();
-    let db = firebase.fb.firestore();
-    db.collection(token)
-      .doc(e.target.value)
-      .update({ datePurchased });
-    console.log(dbItems);
+  // Checks if a purchase date already exists - if not, creates a purchased date and increments # of purchases - if so, also sets the most recent purchase estimate, the most recent purchase interval and the calculated date of the next purchase
+  const handleChange = (e, item) => {
```

Comment:
Awesome! I love that you're passing in the `item` here. This makes it really easy to keep track of which item they selected.